src/drops/token/HolographDropERC721.sol

```diff
@@ -321,6 +324,13 @@ contract HolographDropERC721 is NonReentrant, ERC721H, IHolographDropERC721 {
       });
   }
 
+  /// @notice The Holograph fee is a flat fee for each mint
+  /// @dev Gets the Holograph protocol fee for amount of mints
+  function getHolographFee(uint256 quantity) public returns (uint256 fee) {
+    holographMintFee = 100000; // $0.10 USD (6 decimals)
```

Comment:
@alexanderattar @ACC01ADE I was expecting this to call the price oracle to get the mint fee as this will be PER - network.

Reply:
It calls it in the payoutHolographFee function

Reply:
I would like to move this initialization to another function such as init. Originally I was putting this as a state variable, but was running into issues with this. Would like to discuss during the meeting today

Reply:
Ended up successfully moving this to the init function